src/Symfony/Bridge/ProxyManager/LazyProxy/Instantiator/LazyLoadingValueHolderFactoryV1.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\ProxyManager\LazyProxy\Instantiator;
+
+use ProxyManager\Factory\LazyLoadingValueHolderFactory as BaseFactory;
+use Symfony\Bridge\ProxyManager\LazyProxy\PhpDumper\LazyLoadingValueHolderGenerator;
+
+/**
+ * @internal
+ */
+class LazyLoadingValueHolderFactoryV1 extends BaseFactory
+{
+    private $generatorV1;
```

Comment:
why does it have a different name here ? Except for the property name, the code is the same in v1 and V2.

Reply:
we could just use a name available in both versions